packages/nx-aws-cache/src/tasks-runner/encryptor/Decrypt.ts

```diff
@@ -0,0 +1,70 @@
+
```

Comment:
Please follow file naming convention and use `yarn format`.

Reply:
Not fixed. Naming convention for files is `dash-case`. No upper letters.
Format check still failing for next files:

README.md
e2e/nx-aws-cache-e2e/project.json
packages/nx-aws-cache/src/tasks-runner/encryptor/Encrypt.ts
packages/nx-aws-cache/src/tasks-runner/encryptor/index.ts
packages/nx-aws-cache/src/tasks-runner/runner.ts